src/Spago/Bower.hs

```diff
@@ -31,6 +33,12 @@ bowerPath :: IsString t => t
 bowerPath = "bower.json"
 
 
+bowerCmd :: Text
+bowerCmd = case System.buildOS of
+  Windows -> "bower." -- workaround windows issue: https://github.com/haskell/process/issues/140
```

Comment:
Wow this is cool 😄

Reply:
But still broke :face_with_head_bandage: I'll try and find a Windows VM.